package.json

```diff
@@ -32,9 +32,9 @@
   },
   "homepage": "https://github.com/AlexanderRichey/styled-react-modal#readme",
   "peerDependencies": {
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "styled-components": "^6.1.1"
+    "react": ">=18 <19",
+    "react-dom": ">=18 <19",
+    "styled-components": ">=6 <7"
```

Comment:
There is no [styled-components v7](https://styled-components.com/releases), so I think the previous version is actually correct.

Reply:
Yeah, it's to hint if they have breaking changes on a version upgrade in the future to v7. If they release 7 and it works then it could be changed to <8